cluster-management/setup/cluster-architectures/index.md

```diff
@@ -138,12 +144,13 @@ Look how simple the cloud-config becomes:
 #cloud-config
 
 coreos:
+  etcd2:
+    proxy: on 
+    listen-client-urls: http://localhost:2379
```

Comment:
when you are running proxy mode, you do not need to do this. there was a bug in 2.0.11 and we fixed it in 2.0.12. coreos skipped etcd 2.0.11